features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c), WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import Dialog from '@material-ui/core/Dialog';
+import Button from '@material-ui/core/Button';
+import MUIDataTable from "mui-datatables";
+
+import { FormattedMessage } from 'react-intl';
+import Subscription from 'AppData/Subscription';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        padding: theme.spacing(3, 2),
+    },
+}));
```

Comment:
We can remove this

Reply:
Done : https://github.com/wso2/carbon-apimgt/pull/7306/commits/d20d664c58b0dfde9632423e3db399e740aaadfa